contrib/exec-healthz/Makefile

```diff
@@ -0,0 +1,17 @@
+all: push
+
+# 0.0 shouldn't clobber any released builds
+TAG = 0.0
+PREFIX = gcr.io/google_containers/healthz-server
+
+server: simpleHealthzServer.go
+	CGO_ENABLED=0 GOOS=linux go build -a -installsuffix cgo -ldflags '-w' -o healthz_server ./simpleHealthzServer.go
```

Comment:
Yeah this comes from naming convention i suppose. I'll go with exec_healthz, exec-healthz, exechealthz. 
All directories are '-' separated. Dns rules apply for names of containers. No .go files have _.